### ISSUE TITLE
"No appSettings.json files were found in your project"

### ISSUE THREAD

**Issue by nickpreston24** [jasonswearingen/NotNot.AppSettings#2]
I tried installing your package, then building on a fresh new razor pages project, and this happened:

![image](https://github.com/user-attachments/assets/25b17897-7231-4465-ac72-9774d1eb687e)

I've tried renaming it to camelCase as well.

Thanks.

Machine: Manjaro Linux (latest)
.net ver: 8.0

**Comment by wzuqui** [comment]
Try adding this in .csproj

```xml
<ItemGroup>
  <AdditionalFiles Include="appsettings.json" />
</ItemGroup>
```

**Comment by jasonswearingen** [comment]
yes, as @wzuqui  says (thank you).   also be sure to set your appSettings.json to always copy to the output folder.

I made changes to the error message to be more descriptive, they will be up in the next version today or tomorrow.

FYI I am in the process of moving the sourcecode to here https://github.com/NotNotTech/NotNot-MonoRepo/tree/master/src/nuget/NotNot.AppSettings  I'll properly redirect things/issues in the next few days also.

**Comment by jasonswearingen** [comment]
@nickpreston24 also I apologize for not seeing your raised issue.   I missed it's email notification :(
